src/test/java/de/sakpaas/backend/service/LocationApiSearchDasTest.java

```diff
@@ -0,0 +1,39 @@
+package de.sakpaas.backend.service;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import de.sakpaas.backend.HappyHamsterTest;
+import de.sakpaas.backend.dto.OsmResultLocationListDto;
+import de.sakpaas.backend.util.OsmImportConfiguration;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+class LocationApiSearchDasTest extends HappyHamsterTest {
+
+  final String country = "DE";
+  @Autowired
+  LocationApiSearchDas locationApiSearchDas;
+
+  @Test
+  void testEmptyShoptypeList() {
+    OsmImportConfiguration importConfiguration = new OsmImportConfiguration();
+
+
+    importConfiguration.setShoptypes(new ArrayList<>());
+    List<OsmResultLocationListDto.OsmResultLocationDto> results =
+        locationApiSearchDas.getLocationsForCountry(importConfiguration);
```

Comment:
ich bin ja auch eher ein Fan von integration Tests, aber in dem Fall wird daraus ja wirklich eher ein Gedulds-Test :D

Reply:
Wenn du keine weiteren Tests schreibst @DavidSchmidt00 , musst du diesen Test auf jeden Fall entfernen.

Reply:
waruuum

Reply:
Weil der Test eine Anfrage an OSM macht, oder sehe ich das falsch?
Und das darf auf keinen Fall sein

Reply:
Ne wenn man keine Types reingibt, geht er garnicht erst auf die api. Das sollte ja mehr oder weniger getestet werden